### ISSUE TITLE
[WASM] PolyLineSegment didn't work

### ISSUE THREAD

**Issue by Daoting** [unoplatform/uno#3613]
## Current behavior
Path does not draw content when the content is PolyLineSegment.
```
            PolyLineSegment pls = new PolyLineSegment();
            pls.Points.Add(new Point(10, 10));
            pls.Points.Add(new Point(200, 50));
            pls.Points.Add(new Point(150, 100));
            pls.Points.Add(new Point(10, 150));
            pls.Points.Add(new Point(50, 200));
            pls.Points.Add(new Point(100, 200));

            PathFigure pf = new PathFigure();
            pf.StartPoint = pls.Points[0];
            pf.Segments.Add(pls);
            pf.IsClosed = false;

            var geometry = new PathGeometry();
            geometry.Figures.Add(pf);
            Path path = new Path { Data = geometry, Fill = AtRes.RedBrush, Stroke = AtRes.RedBrush, Width = 300, Height = 300 };
            _con.Content = path;
```
The output html is：
`<svg id="149394" XamlType="Windows.UI.Xaml.Shapes.Path" XamlHandle="149394" class="uno-path uno-arbitraryshapebase uno-shape uno-frameworkelement uno-uielement" tabindex="-1" style="position: absolute; top: 0px; left: 0px; width: 300px; height: 300px;"><path id="149330" XamlType="Windows.UI.Xaml.Wasm.SvgElement" XamlHandle="149330" tabindex="-1" class="uno-svgelement uno-uielement" d="M 10,10" style="fill: rgb(255, 0, 0); stroke-width: 1px; stroke: rgb(255, 0, 0); transform: matrix(1, 0, 0, 1, 0, 0);"></path></svg>`

## Environment

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [x] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

Visual Studio:

- [ ] 2017 (version: )
- [x] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by jeromelaban** [comment]
Thanks for the report. 

Could you indicate which version of Uno you tried this with ?

**Comment by Daoting** [comment]
Uno.UI 2.4.4

**Comment by jeromelaban** [comment]
We've made significant changes in Uno 3.0 dev around this, could you validate using the latest dev package?

**Comment by Daoting** [comment]
The bug still exists after upgrading to V3.0.0-dev.1002. There is a new bug [#3654](https://github.com/unoplatform/uno/issues/3654). @jeromelaban

**Comment by MartinZikmund** [comment]
Fixed by #5100
